Content.Shared/Weapons/Ranged/Systems/SharedGunSystem.cs

```diff
@@ -207,7 +209,8 @@ public void AttemptShoot(EntityUid user, EntityUid gunUid, GunComponent gun, Ent
 
     private void AttemptShoot(EntityUid user, EntityUid gunUid, GunComponent gun)
     {
-        if (gun.FireRate <= 0f)
+        if (gun.FireRate <= 0f ||
+            _mobState.IsIncapacitated(user))
```

Comment:
It would probably catch more edge cases if this call was replaced with ActionBlockerSystem's `CanUseHeldEntity`.